net-proxy/trojan/files/trojan.initd

```diff
@@ -0,0 +1,35 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+TJ_CONFIG="/etc/trojan/config.json"
+TJ_PIDFILE="/run/trojan.pid"
+TJ_LOG="/var/log/trojan.log"
+
+depend() {
+	need net
+}
+
+checkconfig() {
+	if [ ! -f ${TJ_CONFIG} ]; then
+		ewarn "${TJ_CONFIG} does not exist."
+	fi
+}
+
+start() {
+	checkconfig || return 1
+
+	ebegin "Starting Trojan"
+	ebegin "Log File: ${TJ_LOG}"
```

Comment:
You can't ebegin twice without eend. I suppose you want to print the log file after eend.